rio_cogeo/cogeo.py

```diff
@@ -16,6 +16,7 @@
 from rasterio.enums import Resampling as ResamplingEnums
 from rasterio.env import GDALVersion
 from rasterio.io import DatasetReader, DatasetWriter, MemoryFile
+from rasterio.path import parse_path
```

Comment:
👀

Reply:
rasterio is a bit weird because it has its own Path class (mostly converting between /vis* -> url. This was created before pathlib existed 🤷‍♂️.